src/extras/PMREMGenerator.js

```diff
@@ -107,9 +107,11 @@ class PMREMGenerator {
 	 * @param {number} sigma
 	 * @param {number} near
 	 * @param {number} far
+	 * @param {Number} size
+	 * @param {Number} position
 	 * @return {WebGLRenderTarget}
 	 */
-	fromScene( scene, sigma = 0, near = 0.1, far = 100 ) {
+	fromScene( scene, sigma = 0, near = 0.1, far = 100, size = 256, position = new Vector3( 0, 0, 0 ) ) {
```

Comment:
Couldn't we have these properties in class scope? e.g:
```js
const sceneRT = new THREE.PMREMGenerator( renderer );
sceneRT.position = new THREE.Vector3( ... );
sceneRT.faceSize = 256;
sceneRT.fromScene( ... )
```
Maybe we will have more options in the future, that should help.

Reply:
`position` and `size` are only relevant for `fromScene()` and have no purpose for the rest of the API. Hence, I would favor to keep both in the method signature. As a compromise, how about doing the following?
```js
generator. fromScene( scene, sigma, near, far, options );
```
In this way, we can add `position`, `size` and potentially future configuration properties to `options`.

Reply:
Sounds good to me.